layouts/_partials/components/codeblock.html

```diff
@@ -10,7 +10,15 @@
 
       {{- $base_url = strings.TrimSuffix "/" $base_url -}}
       {{- $filename = strings.TrimPrefix "/" $filename -}}
-      {{- $file_url := urls.JoinPath $base_url $filename -}}
+
+      {{- $file_url := $filename -}} <!-- default to just filename -->
```

Comment:
```suggestion
      {{- /* Default to just filename */ -}}
      {{- $file_url := $filename -}}
```

It's better to use template comments instead of HTML comments.

Reply:
ha, same thought here, you were quicker!